### DISCUSSION TITLE
Is it possible to copy the "Skia.GTK" folder that is made on Windows (Net 6/ VS 2022) and Have it run on MacOS

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#8263] [Q&A]
Greetings,
  I've seen this question asked on the discord for the older template (<net6 , VS2019) but it was never answered so I want to throw it here. Is it possible to run the App.Skia.Gtk.dll file composed in Windows and run it by using dotnet App.Skia.Gtk.dll because this seems to keep occurring even with Homebrew gtk+3 successfully installed :

      System.DllNotFoundException: libgtk-3-0.dll, libgtk.so.0, libgtk-3.0.dylib, gtk-3.dll

**Comment by jeromelaban** [comment]
Homebrew installation should be enough to get the application running. Does it happen when building the application directly on the mac?

**Reply by bstrother1** [reply]
> Homebrew installation should be enough to get the application running. Does it happen when building the application directly on the mac?

Yes, it can't see it when built in Visual Studios 2022 either. Following along with https://github.com/trustable-code/NiGui/issues/63  I changed the ARM64 Homebrew to x86_64 still no dice. Running brew info gtk+3 

gtk+3: stable 3.24.33 (bottled)

Toolkit for creating graphical user interfaces

https://gtk.org/

/usr/local/Cellar/gtk+3/3.24.33 (723 files, 52.8MB) *

  Poured from bottle on 2022-03-07 at 14:49:38

From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/gtk+3.rb

License: LGPL-2.0-or-later

==> Dependencies

Build: docbook ✘, docbook-xsl ✘, gobject-introspection ✔, meson ✘, ninja ✘, pkg-config ✔

Required: atk ✔, gdk-pixbuf ✔, glib ✔, gsettings-desktop-schemas ✔, hicolor-icon-theme ✔, libepoxy ✔, pango ✔

==> Analytics

install: 30,907 (30 days), 65,931 (90 days), 282,209 (365 days)

install-on-request: 15,315 (30 days), 33,138 (90 days), 143,564 (365 days)

build-error: 17 (30 days)

**Reply by bstrother1** [reply]
running lipo -f /usr/local/Cellar/gtk+3/3.24.30/lib/libgtk-3.0.dylib : Non-fat file: /usr/local/Cellar/gtk+3/3.24.33/lib/libgtk-3.0.dylib is architecture: x86_64

**Reply by bstrother1** [reply]
linked to incorrect ticket https://github.com/GtkSharp/GtkSharp/issues/249

**Reply by bstrother1** [reply]
Looking at what gtk+3 has installed by running normally:
<img width="684" alt="Screen Shot 2022-03-08 at 10 12 38 AM" src="https://user-images.githubusercontent.com/19878907/157266694-9fc86a51-b471-479b-81aa-0c7988fb5051.png">

Which doesn't match up with what is being requested by Uno, which is:
System.DllNotFoundException: Gtk: libgtk-3-0.dll, libgtk-3.so.0, libgtk-3.0.dylib, gtk-3.dll

Am I not understanding something or is this a compatibility issue with Uno?

**Reply by bstrother1** [reply]
Could this be related to the fix [[ ](https://gitlab.gnome.org/GNOME/gtk/-/issues/4342)](https://gitlab.gnome.org/GNOME/gtk/-/issues/4342)
